edit-post/keyboard-shortcuts.js

```diff
@@ -1,9 +1,15 @@
-const isMac = window.navigator.platform.toUpperCase().indexOf( 'MAC' ) >= 0;
-const mod = isMac ? '⌘' : 'Ctrl';
+/**
+ * Internal dependencies
+ */
+import keyboardShortcut, {
+	ALT,
+	CONTROL,
+	SHIFT,
+} from 'utils/keyboard-shortcut';
 
 export default {
 	toggleEditorMode: {
 		value: 'mod+shift+alt+m',
-		label: `${ mod }+Shift+Alt+M`,
+		label: keyboardShortcut(CONTROL, SHIFT, ALT, 'M'),
```

Comment:
Unrelated to this PR, but this one should probably change the a standard `access` combination for WordPress.